app/assets/stylesheets/fae/modules/tables/_actions.scss

```diff
@@ -7,6 +7,10 @@ th {
     width: 120px
   }
 
+  &.-action-narrow {
```

Comment:
I had to add it for this ticket, it will be added to the new admin for that one big site - this is for datetime. I'm happy to document it if there is a good place for it.